book-2-platform-api/chapters/API_MODELS_CONTROLLERS.md

```diff
@@ -96,70 +158,121 @@ namespace CoffeeShop.Controllers
         [HttpGet]
         public async Task<IActionResult> Get()
         {
-            using (IDbConnection conn = Connection)
+            using (SqlConnection conn = Connection)
```

Comment:
are we not using `var` because its helpful for them to see the types?

Reply:
I avoid using `var` in example code so that they get used to see types as much as possible.

Reply:
do you find that this causes them to not use var in real code?

Reply:
It's hit and miss. High acceleration students get type inference quickly and will switch to var fairly soon. Low acceleration students will stick with strict typing longer.

Reply:
gotcha.  I only show explicit typing in the first code along.  After that its implicit for things with assignment on the right, type for things without.  I guess I always figured method signatures and properties and fields were enough to drive types home.  I also explain the hovering over variables will tell them what the implicit type is.